src/page/bank-payment.page.ts

```diff
@@ -4,7 +4,8 @@ export class BankPaymentPage {
   private confirmOrderBtn: ElementFinder;
 
   constructor () {
-    this.confirmOrderBtn = $('#cart_navigation > button > span');
+    // we can remove span
+    this.confirmOrderBtn = $('#cart_navigation button');
```

Comment:
I think that this doesn't make any difference with the original one, you can play a little bit more, try to use a different locator, for example, the text related with the button seems to be unique "I confirm my order"". Find a locator in where you can use that text.